xdsl/dialects/llvm.py

```diff
@@ -105,6 +105,43 @@ def is_typed(self):
         return not isinstance(self.type, NoneAttr)
 
 
+@irdl_attr_definition
+class LLVMArrayType(ParametrizedAttribute, MLIRType):
+    name = "llvm.array"
+
+    type: ParameterDef[Attribute]
+    size: ParameterDef[IntAttr]
```

Comment:
I would reorder them to stay consistent with the printed IR. Not a big issue though :)

Reply:
Done :)